security/pkg/server/ca/authenticate/cert_authenticator.go

```diff
@@ -43,7 +43,17 @@ func (cca *ClientCertAuthenticator) AuthenticatorType() string {
 // method assumes that certificate chain has been properly validated before
 // this method is called. In other words, this method does not do certificate
 // chain validation itself.
-func (cca *ClientCertAuthenticator) Authenticate(ctx context.Context) (*security.Caller, error) {
+func (cca *ClientCertAuthenticator) Authenticate(authReq security.AuthenticateContext) (*security.Caller, error) {
```

Comment:
nit authReq -> authCtx. Same in other places

Reply:
SG